tutorials/tour/_posts/2017-02-13-higher-order-functions.md

```diff
@@ -9,34 +9,112 @@ categories: tour
 num: 8
 next-page: nested-functions
 previous-page: anonymous-function-syntax
+assumed-knowledge: sequence-comprehensions, anonymous-function-syntax
 ---
 
-Scala allows the definition of higher-order functions. These are functions that _take other functions as parameters_, or whose _result is a function_. Here is a function `apply` which takes another function `f` and a value `v` and applies function `f` to `v`:
-
+Higher order functions take other functions as parameters or return a function as
+a result. This is possible because functions are first-class values in Scala.
+One of the most common examples is the higher-order
```

Comment:
Hmm.. maybe I'm missing something but just so it's clear `One of the most common examples` *with* the `s` is the correct English.

That said, you can certainly say `A common example...`